runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineJob.java

```diff
@@ -167,20 +179,19 @@ public DataflowPipelineJob getReplacedByJob() {
    * @throws InterruptedException
    */
   @Nullable
-  public State waitToFinish(
-      long timeToWait,
-      TimeUnit timeUnit,
-      MonitoringUtil.JobMessagesHandler messageHandler)
-          throws IOException, InterruptedException {
-    return waitToFinish(timeToWait, timeUnit, messageHandler, Sleeper.DEFAULT, NanoClock.SYSTEM);
+  @VisibleForTesting
+  public State waitUntilFinish(
```

Comment:
Cannot reduce its visibility.
This is used in TestDataflowRunner in testing package.